etc/sdk.api.md

```diff
@@ -170,10 +170,10 @@ export class GundbMetadataStore implements UserMetadataStore {
     findBucket(bucketSlug: string): Promise<BucketMetadata | undefined>;
     findFileMetadata(bucketSlug: string, dbId: string, path: string): Promise<FileMetadata | undefined>;
     findFileMetadataByUuid(uuid: string): Promise<FileMetadata | undefined>;
-    // Warning: (ae-forgotten-export) The symbol "GunChainReference" needs to be exported by the entry point index.d.ts
-    // Warning: (ae-forgotten-export) The symbol "GunDataState" needs to be exported by the entry point index.d.ts
-    static fromIdentity(username: string, userpass: string, gunOrServer?: GunChainReference<GunDataState> | string): Promise<GundbMetadataStore>;
+    // Warning: (ae-forgotten-export) The symbol "GunInit" needs to be exported by the entry point index.d.ts
+    static fromIdentity(username: string, userpass: string, gunOrServer?: GunInit | string): Promise<GundbMetadataStore>;

```

Comment:
This was there from before but just wondering now, if we should name it like `MetadataStoreFromIdentity`?

Reply:
The `fromIdentity()` is a static method is already on the `MetadataStore`, so it may look weird if the user has too do a `Metadatastore.metadataStoreFromIdentity()`

Reply:
Or do you mean we should have a standalone none static factory function just `metadataStoreFromIdentity()` that is not attached to an class/object?

Reply:
Ah no your right, it's fine the way it is.